### PR TITLE
#28 - design of companies site

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
image
image
image
image

Mobile:
image
image
image
image
[... companies]
image

Optional, if links to social medias are available, twitter and fb icons will be also rendered, like below:
image

### CODE REVIEW

File: frontend/components/Companies/InvitationToJoin.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="absolute inset-0">
+      <img class="h-full w-full object-cover" src="/images/lpm_22/lpm22-img1.webp" alt="">
+      <div class="absolute inset-0 bg-zinc-800 mix-blend-multiply" aria-hidden="true"/>
+    </div>
+    <div class="relative mx-auto max-w-7xl py-24 px-4 sm:py-32 sm:px-6 lg:px-8">
+      <div class="mx-auto max-w-7xl py-8 px-4 text-center sm:px-6 lg:py-10 lg:px-8">
+        <h2 class="tracking-tight text-white">
+          <span class="mb-7 block text-3xl">Wypromuj swoją firmę!</span>
+          <span class="block text-2xl">
+            Chciałbyś, aby Twoja firma brała udział w brandowych spotkaniach? <br>A może chcesz na stałe dołączyć do zespołu
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Companies/InvitationToJoin.vue]:
```suggestion
            Chcesz, aby Twoja firma brała udział w brandowych spotkaniach? <br>A może masz chęć na stałe dołączyć do zespołu 
```

Comment (EwelinaSkrzypacz) [reply frontend/components/Companies/InvitationToJoin.vue]: Also I think that I'll remove `<br>` tag and use `<span>` or `<div>` to wrap this text.

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Please add note in a `readme` about copying `.env.example` file.

`cp .env.example .env`

File: readme.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ Or you can just run:
 make setup
 ```
 
+Copy .env.example file and change it name to ".env":
```
Comment (EwelinaSkrzypacz) [inline readme.md]:
```suggestion
Copy `.env.example` file and change it name to `.env`:
```

File: readme.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ Or you can just run:
...
+```bash
+cp .env.example .env
+```
+Add your token to VITE_VUE_APP_MAPBOX_TOKEN in .env
```
Comment (EwelinaSkrzypacz) [inline readme.md]:
```suggestion
Add token to `VITE_VUE_APP_MAPBOX_TOKEN` in `.env`
```

File: .env.example [hunk 4]
```diff
@@ -0,0 +1 @@
+VITE_VUE_APP_MAPBOX_TOKEN="YOUR TOKEN"
```
Comment (EwelinaSkrzypacz) [inline .env.example]:
```suggestion
VITE_VUE_APP_MAPBOX_TOKEN=
```

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**